apis/apiextensions/fn/proto/v1/run_function.proto

```diff
@@ -223,9 +223,6 @@ message Resource {
   //
   // * A Function should set this field to READY_TRUE in a RunFunctionResponse
   //   to indicate that a desired composed resource is ready.
-  //
```

Comment:
If we make this change we should document (here and elsewhere) the semantics. Which I think would be:

* If you omit this field, we'll derive XR readiness from composed resource readiness
* If you specify this field, we'll ignore composed resource readiness and use what you explicitly told us